lib/plugins/features/movies/index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Controller     = require('./controller');
+const MovieValidator = require('../../../validators/movie');
+
+exports.register = (server, options, next) => {
+
+  server.route([{
+    method: 'POST',
+    path: '/movies',
+    config: {
+      handler: (request, reply) => {
```

Comment:
Since this is a 1 line controller, we can also use short hand here.
```js
handler: (request, reply) => reply(Controller.create(request.payload)),
```